models/index.js

```diff
@@ -16,10 +16,22 @@ exports.selectArticleById = (article_id) => {
   });
 };
 
-exports.selectAllArticles = () => {
-  const queryStr = `SELECT articles.author, title, articles.article_id, topic, articles.created_at, article_img_url, COUNT(comments.article_id) AS comment_count, SUM(comments.votes) AS votes FROM articles JOIN comments ON articles.article_id=comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at;`;
+exports.selectAllArticles = (query) => {
+  if (Object.keys(query)[0] && ![ 'topic' ].includes(Object.keys(query)[0])) {
```

Comment:
I would consider a more readable way of white listing your query here. You may need to check that a topic exists in the database rather than just check that topic is included in the query keys.